docs/architecture/deep-dives/passkeys/index.md

```diff
@@ -0,0 +1,7 @@
+# FIDO2 and Passkeys
+
+This is not a comprehensive guide to FIDO2, but a summary of the standard's crucial aspects for
+working within the Bitwarden context.
+
+It is not intended to be a comprehensive guide to FIDO2, but rather a summary of the most important
```

Comment:
⛏️ This appears to be largely duplicative between the two paragraphs.

Reply:
I think I forgot to remove the second one. Fixed!